doc/source/PreConfiguredSites.rst

```diff
@@ -118,16 +118,30 @@ NASA Discover SCU16
 
 The following is required for building new spack environments with any supported compiler on this platform.
 
-**NEEDS UPDATING**
-
 .. code-block:: console
 
    module purge
    module use /discover/swdev/gmao_SIteam/modulefiles-SLES12
    module use /discover/swdev/jcsda/spack-stack/scu16/modulefiles
    module load miniconda/3.9.7
-   module load ecflow/5.8.4
 
+For ``spack-stack-1.8.0`` with Intel, load the following modules after loading miniconda:
```

Comment:
This doesn't go into readthedocs anymore, see nautilus/narwhal - instead, please update https://github.com/JCSDA/spack-stack/wiki/spack%E2%80%90stack%E2%80%901.8.0-release-documentation